src/frontend/komponenter/Behandling/Aktivitet/Aktivitet/Aktivitet.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { VilkårProps } from '../../Inngangsvilkår/vilkårprops';
+import { AktivitetsvilkårType } from '../../Inngangsvilkår/vilkår';
+import Vilkår from '../../../Felleskomponenter/Vilkår';
+import VisEllerEndreVurdering from '../../Vurdering/VisEllerEndreVurdering';
+import AktivitetInfo from './AktivitetInfo';
+
+export const Aktivitet: React.FC<VilkårProps> = ({
+    vurderinger,
+    grunnlag,
+    lagreVurdering,
+    nullstillVurdering,
+    feilmeldinger,
+}) => {
+    const vurdering = vurderinger.find((v) => v.vilkårType === AktivitetsvilkårType.AKTIVITET);
+    if (!vurdering) {
+        return <div>Mangler vurdering for aktivitet</div>;
+    }
+    return (
+        <Vilkår
+            vilkårtittel={{
+                tittel: 'Aktivitet',
+                vilkårsresultat: vurdering.resultat,
+            }}
+        >
+            {{
+                left: <AktivitetInfo aktivitet={grunnlag.aktivitet} />,
+                right: (
```

Comment:
Liker denne måten å legge inn venstre/høyre kolonne på, men kalle det venstre/høyre for å holde oss til norsk?

Reply:
Yes 👍